pyproject.toml

```diff
@@ -367,6 +424,9 @@ lint.ignore = [  # NOTE: non-permanent exclusions should be added to `.ruff.toml
     # flake8-fixme (FIX)
     "FIX002",  # Line contains TODO | notes for improvements are OK iff the code works
 
+    # flake8-implicit-str-concat (ISC)
+    "ISC001",  # Implicitly concatenated string literals on one line
```

Comment:
I don't see where this is coming from. Didn't we apply this rule before ?

Reply:
Removed.